crmsh.spec.in

```diff
@@ -179,7 +160,7 @@ touch -r doc/crm.8.adoc{.timestamp,}
 make %{_smp_mflags} VERSION="%{version}" sysconfdir=%{_sysconfdir} localstatedir=%{_var}
 
 %if %{with regression_tests}
-tox -q
+./test/run --quiet
```

Comment:
Why have you updated this? Wasn't better to use tox?

Reply:
Honestly I didn't notice this, just copy from NHF's spec file.
I will take a look at it. Thanks for pointing out!

Reply:
@arbulu89 , Already changed to `tox`

Reply:
If this PR is ready, I will merge it when your CI's change for `spec.in` file is ready

Reply:
@liangxin1300 Perfect. I will let you know when the CI project is updated and working. Wait meanwhile.